prog/vnet/update_load_balancer_node.rb

```diff
@@ -129,22 +131,30 @@ def generate_lb_ip_set_definition(neighbor_vms)
       "elements = {#{neighbor_vms.map(&:private_ipv6).join(", ")}}"]
   end
 
-  def generate_lb_map_defs_ipv4(current_port)
-    load_balancer.active_vm_ports
+  def generate_lb_map_defs(current_port, address_selector)
+    items = load_balancer.active_vm_ports
```

Comment:
Likewise, in this method, you want to eagerly load the associations:

```ruby
load_balancer.active_vm_ports{|ds| ds.eager(:load_balancer_port, load_balancer_vm: :vm)}
```

You may not need the `vm` association eagerly loaded if you change from `vm_port.load_balancer_vm.vm.id` to `vm_port.load_balancer_vm.vm_id`.